fluster/test_suite.py

```diff
@@ -287,23 +283,65 @@ def _run_worker(self, test: Test) -> TestVector:
 
         return test.test_vector
 
+    def _get_max_length_list_name(self, _list: List[str], name: str) -> int:
+        max_length = len(name)
+        for elem in _list:
+            length = len(elem)
+            max_length = max(max_length, length)
+        return max_length
+
+    @lru_cache(maxsize=None)
+    def _get_result_line(
+        self,
+        test_suite_text: str,
+        decoder_text: str,
+        test_vector_text: str,
+        result_text: str,
+        decoder_name: Optional[str] = None,
+    ) -> str:
+        decoder_name = decoder_text if not decoder_name else decoder_name
+        tests_suite_max_len = self._get_max_length_list_name(
+            [self.name], TestSuite.TEST_SUITE_NAME
+        )
+        decoder_max_len = self._get_max_length_list_name(
+            [decoder_name], TestSuite.DECODER_NAME
+        )
+        test_vectors_max_len = self._get_max_length_list_name(
+            list(self.test_vectors.keys()), TestSuite.TEST_VECTOR_NAME
+        )
+
+        return (
+            f"[{test_suite_text:{tests_suite_max_len}}] ({decoder_text:{decoder_max_len}}) "
+            f"{test_vector_text:{test_vectors_max_len}} ... {result_text}"
+        )
+
     def run_test_suite_in_parallel(
         self, jobs: int, tests: List[Test], failfast: bool
     ) -> None:
         """Run the test suite in parallel"""
         test_vector_results: List[TestVector] = []
-        max_len = self._get_max_length_test_vectors_name()
+        decoder = tests[0].decoder
+
         print(
-            f'[TEST_SUITE]\t(DECODER)\t{"TEST_VECTOR":{max_len}} ... RESULT\n{"-" * 70}'
+            self._get_result_line(
+                TestSuite.TEST_SUITE_NAME,
+                TestSuite.DECODER_NAME,
+                TestSuite.TEST_VECTOR_NAME,
+                "RESULT",
```

Comment:
Since you've already defined the rest of the arguments in "global" variables it's worth it to define "RESULT" too for uniformity.

```suggestion
                TestSuite.RESULT,
```

Reply:
You're right. Fixed!